src/contact_load.js

```diff
@@ -0,0 +1,9 @@
+export default function contactLoad(content) {
+  content.innerHTML = `<div class='contact-page'>
+  <h1 class='contact'>Get in touch:</h1>
+  <p class='info'>Whatsapp</p>
+  <p class='info'>Intagram</p>
+  <p class='info'>Phone</p>
+  <p class='info'>Email</p>
+  </div>`;
+}
```

Comment:
- [ ] [REQUIRED] This project should use arrow functions instead of the `function` keyword.
![image](https://user-images.githubusercontent.com/73254040/100486169-82253100-30c8-11eb-9515-52e5bbcd0f11.png)
[Source](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622983/)